pandas/tests/io/test_fsspec.py

```diff
@@ -289,7 +291,21 @@ def test_stata_options(fsspectest):
 
 
 @td.skip_if_no("tabulate")
-def test_markdown_options(fsspectest):
+def test_markdown_options(request, fsspectest):
```

Comment:
use ``@pytest.importorskip`` here with minversion?(pretty sure the issue here is with fsspec's min version only, but not sure).

Reply:
With `@pytest.importorskip` I think I would have to specify the min version + 1 to guarantee skipping the min version. I think the +1 aspect isn't as simple as adding +1 to whatever comes out of `__version__`